tlumip/tlumip
# Adding ability to run SWIMR at the end of a model run using tsteps.csv

Changes to dependencies64.zip includes additional R libraries needed for swimr and updates to the following previously included R libraries:

| Package | Old | New |
|---------- | ----|-------|
| Rcpp | 0.12.7 |  0.12.16 |
| stringi | 1.1.2  | 1.1.7 |
| stringr | 1.1.0  | 1.3.0 |
| swimr | 0.2.7 |  0.2.8 |

See also R-requirements_old_and_new.csv

Addresses #95 

To test these changes: 

1) checkout this PR as a local branch

```
git checkout -b landisrm-master master
git pull https://github.com/landisrm/tlumip.git master
```
2) Then modify tsteps.csv to indicate that you want to run the SWIMR_reference and SWIMR_compare templates in year 50 (and nothing else).

3) Modify globalTemplate.properties to point to desired VIZ db files for the SWIMR output.

4) Run build_run.bat and run_model.bat.  

5) Check for HTML files in the same directory as the "current" db file.  

6) Check model_report.txt and model_run_output. txt
